app/main/views/g_cloud.py

```diff
@@ -337,15 +337,16 @@ def project_create():
     flash(PROJECT_CREATED_MESSAGE, 'success')
 
     return redirect(url_for('.view_project',
+                            framework_framework=framework_framework,
                             project_id=project['id']
                             ))
 
 
-@direct_award.route('/projects/<int:project_id>', methods=['GET'])
-def view_project(project_id):
+@direct_award.route('/<string:framework_framework>/projects/<int:project_id>', methods=['GET'])
```

Comment:
Yep this `framework_framework` works here I think.

Same pattern as we already have for, e.g.
```
/<framework_framework>/opportunities
/<framework_framework>/opportunities/<brief_id>
```